api/src/cli/commands/security/key.ts

```diff
@@ -0,0 +1,30 @@
+import { readFileSync, existsSync, writeFileSync } from 'fs';
+import { join as pathJoin } from 'path';
+import { v4 as uuidv4 } from 'uuid';
+import logger from '../../../logger';
+
+export default async function generateKey({ show }: { show?: boolean }): Promise<void> {
+	const key = uuidv4();
+
+	if (show) {
+		process.stdout.write(`Your new app key: ${key}\n`);
+
+		process.exit(0);
+	}
+
+	const rootPath = process.cwd();
+	const envPath = pathJoin(rootPath, '.env');
+
+	if (!existsSync(envPath)) {
+		logger.error(`No .env file found at ${envPath}`);
+		process.exit(1);
+	}
+
+	const envFileContent = readFileSync(envPath)
+		.toString()
+		.replace(/^KEY=(.*)$/gm, `KEY="${key}"`);
```

Comment:
Uhm, for quotes/not-quotes it's not really a problem, here we just replace the line starting with `KEY=`, so it matches `KEY="whatever"` and `KEY=whatever`. About YAML/JSON maybe we can detect it (?) or just let user choose the format with `--yaml` or `--json`? For `docker-compose` they should use `--show`...

...Weeeeell, ok, I guess it's better to just leave the `--show` behaviour as default haha

Reply:
> here we just replace the line starting with KEY=, so it matches KEY="whatever" and KEY=whatever

True, but right now you're replacing the whole line with the quotes version 🙂 I just recall there was at least one environment where the difference in quotes mattered, it might've been Docker though though, not the .env file. 

We could make it work for JSON/YAML, but that doesn't solve it for the function-based config setup, which allows the user to dynamically generate the config in an async function on start 🤔

Reply:
Yeah, you convinced me 👀 I'm going to push the edit soon